packages/components/src/components/option/option.stories.ts

```diff
@@ -34,149 +35,147 @@ export const Default = {
-            },
-            {
-              type: 'slot',
-              name: 'right',
-              value: '<sd-icon slot="right" library="global-resources" name="system/picture"></sd-icon>'
-            }
-          ],
-          args
-        })
-      )}
-    `;
-  }
+export const Selected = {
+  render: () =>
+    html`<div class="flex flex-row gap-12">
+      <sd-select
+        class="h-[160px] w-[420px]"
+        label="Selected option"
+        placeholder="Please Select"
+        placement="bottom"
+        size="lg"
+        max-options-visible="3"
+      >
+        <sd-option value="option-1">Selected Option</sd-option>
+      </sd-select>
+      <sd-select
+        class="h-[160px] w-[420px]"
+        label="Multiple selected option"
+        placeholder="Please Select"
+        placement="bottom"
+        size="lg"
+        max-options-visible="3"
+        multiple
+      >
+        <sd-option value="option-1" checkbox>Selected Checkbox Option</sd-option>
+      </sd-select>
+    </div>`
 };
 
 /**
- * Use the `base`, `label`, `left`, `right` part selectors to customize the option.
+ * Use the `disabled` attribute to disable an option.
  */
 
-export const Parts = {
-  parameters: {
-    controls: {
-      exclude: ['base', 'label', 'left', 'right']
-    }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'template',
-          name: 'sd-option::part(...){outline: solid 2px red}',
-          values: ['base', 'label', 'left', 'right'].map(part => {
-            return {
-              title: part,
-              value: `<style>#part-${part} sd-option::part(${part}){outline: solid 2px red}</style><div id="part-${part}">%TEMPLATE%</div>`
-            };
-          })
-        }
-      },
-      constants: [
-        {
-          type: 'slot',
-          name: 'left',
-          value: '<sd-icon slot="left" library="global-resources" name="system/picture"></sd-icon>'
-        },
-        {
-          type: 'slot',
-          name: 'default',
-          value: 'Option'
-        },
-        {
-          type: 'slot',
-          name: 'right',
-          value: '<sd-icon slot="right" library="global-resources" name="system/picture"></sd-icon>'
-        }
-      ],
-      args
-    });
-  }
+export const Disabled = {
+  render: () =>
+    html`<div class="flex flex-row gap-12">
+      <sd-select
+        class="h-[260px] w-[420px]"
+        label="Disabled Options"
+        placeholder="Please Select"
+        placement="bottom"
+        max-options-visible="3"
+      >
+        <sd-option value="option-1" disabled>Disabled Option 1</sd-option>
+        <sd-option value="option-2" disabled>Disabled Option 2</sd-option>
+        <sd-option value="option-3" disabled>Disabled Option 3</sd-option>
+      </sd-select>
+      <sd-select
+        class="h-[160px] w-[420px]"
+        label="Disabled Checkbox Options"
+        placeholder="Please Select"
```

Comment:
Is the text color text.neutral-700 for placeholder text? If not pls open up a bug issue for that. THX

Reply:
https://github.com/orgs/solid-design-system/projects/1/views/1?pane=issue&itemId=78377719